tests/functional/adapter/list_relations_tests/test_show_objects.py

```diff
@@ -73,7 +73,7 @@ def list_relations_without_caching(project) -> List[SnowflakeRelation]:
             database=project.database, schema=project.test_schema, identifier=""
         )
         with get_connection(my_adapter):
-            relations = my_adapter.list_relations_without_caching(schema)
+            relations = my_adapter.list_relations_without_caching(schema.path.schema)
```

Comment:
The original value here was `DATABASE.SCHEMA.` so I just used the `SCHEMA` string itself

Reply:
Was this causing test failures? The type on this argument is `SnowflakeRelation`, not `str`, in `SnowflakeAdapter`.

Reply:
It was causing errors.

Reply:
Well I reverted this because it's working again. I must have introduced a bug way up the chain.